index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mobile-Main</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!-- Header -->
+    <header class="header">
+      <a href="#" class="logo">
+        <h3>My Logo</h3>
+      </a>
+      <button class="menu_btn">
+        <div class="menu">
+          <img src="images/menu.png" alt="" />
+        </div>
+      </button>
+    </header>
+
+    <!-- Headline Section -->
+    <section class="headline">
+      <div class="container">
+        <div class="title">
+          <h1>I’m Mudassir Glad to see you!</h1>
+        </div>
+        <div class="info">
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+        <div class="social_media"><a href="#">LET’S CONNECT</a></div>
```

Comment:
Kindly adjust the color and design of the LET'S CONNECT button to match that of the design template. It has no underline in the Figma design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/198837406-6fc797e7-fc18-4861-b56a-b17de011d8e1.png)

Figma deisgn:
![image](https://user-images.githubusercontent.com/20554583/198837425-6ab95bfc-4809-42b2-871e-1d6476f8a3fe.png)